Readme.md

```diff
@@ -1,3 +1,9 @@
+[![Gem Version](https://badge.fury.io/rb/ar_after_transaction.png)](http://badge.fury.io/rb/ar_after_transaction)
+[![Travis Build Status](https://secure.travis-ci.org/evolve75/RubyTree.png)](http://travis-ci.org/grosser/ar_after_transaction)
+[![Dependency Status](https://gemnasium.com/evolve75/ar_after_transaction.png)](https://gemnasium.com/grosser/ar_after_transaction)
+[![Code Climate](https://codeclimate.com/github/evolve75/RubyTree.png)](https://codeclimate.com/github/grosser/ar_after_transaction)
```

Comment:
Ok, I'll get rid of it. I think that it can be useful for some codebases but I haven't thought much about whether this is one of them.